symphony/bdk/core/auth/auth_session.py

```diff
@@ -0,0 +1,30 @@
+class AuthSession:
+    """RSA Authentication session handle.
+    """
+
+    def __init__(self, authenticator):
+        self._session_token = None
+        self._key_manager_token = None
+        self._authenticator = authenticator
```

Comment:
@symphony-elias Please correct me if I am wrong.

We decided to remove all 'private' attributes. 

For a **single underscore** is merely an agreed-upon convention and does not impose any restrictions on accessing the value of an attribute.

For a **double underscore**, Python mangles these names and it is used to avoid name clashes with names defined by subclasses.

Reply:
It depends, we will use single underscore for private attributes if there is a specific logic for getters or setters (in which case, we will use single underscore for the attribute + properties)